qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,108 @@
+Feature: 1.1.2 Send_CV.feature
+
+Background: 
+	Given user accessed Quantori Career Portal from his browser
+	And user scrolled down to CV sending form
+
+Scenario: 1.1.2_01 User can successfully send CV
+	Given user inputted "Ivan Ivanov" in field "Name"
+	And user inputted "ivan.ivanov@gmail.com" in field "Email"
+	And user attached CV with file format being PDF/DOC/DOCX/RTF with size less than 10 Mb
+	And user checked Checkbox
+	When user clicks Send button
+	Then message box "Thanks for being awesome! We have received your message and will come back to you as soon as we process it." should appear
+	And form should be empty
+
+Scenario: 1.1.2_02 User can't send empty form
+	When user clicks Send button
+	Then all required fields should be highlighted in red
+	And messages "This field is required" should appear under fields "Name", "Email"
+	And button "Attach CV" should be highlighted in red
+	And Checkbox's description "I agree to the personal data processing" should be highlighted in red
+
+
+Scenario: 1.1.2_03 User fills field "Name" with minimum characters
```

Comment:
It is necessary to add a positive test that the "Name" field can contain punctuation (., - ') and Cyrillic characters.